apps/liquidity-provision-dashboard/.env.mainnet

```diff
@@ -2,7 +2,7 @@
 NX_VEGA_CONFIG_URL=https://static.vega.xyz/assets/mainnet-network.json
 NX_VEGA_URL=https://api.vega.xyz/query
 NX_VEGA_ENV=MAINNET
-NX_VEGA_NETWORKS='{\"MAINNET\":\"https://alpha.console.vega.xyz\"}'
+NX_VEGA_NETWORKS={\"MAINNET\":\"https://alpha.console.vega.xyz\",\"TESTNET\":\"https://console.fairground.wtf\",\"STAGNET1\":\"https://stagnet1.console.vega.xyz\",\"STAGNET3\":\"https://stagnet3.console.vega.xyz\"}
```

Comment:
chore (blocking): we do not want to point people to stagnet3 or stagnet1 on mainnet. These are internal test networks we do not wish to publicise.

Reply:
Just fairground and mainnet please

Reply:
That makes a lot of sense now that you say it 🤦‍♀️ thank you!